setup.py

```diff
@@ -20,6 +20,7 @@
 install_requires = [
     "munch",
     "requests",
+    "Click",  # for the CLI and GH Action
```

Comment:
I'd rather not use click to avoid the unnecessary dependency. This script probably won't grow much larger, so `argparse` would IMHO be just fine. But if you prefer click, we can keep it.